File: TemplateMongo/TemplateMongo/Model/WorkEvent.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace TemplateMongo.Model { 
    public class WorkEvent
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }

        [BsonElement("workDescription")]
        public string workDescription { get; set; }
        public RepairType repairType { get; set; }
        public int amount { get; set; }
        public int location { get; set; }

    }
}

File: client/src/app/model/workEvent.ts
import { Strings } from '../strings';
import { RepairType } from './repairType';

export class WorkEvent {
public Id:string;
public workDescription:string;
public repairType:RepairType;
public amount:number;
public location:number;
// public item:string;

/**
 *
 */
constructor() {
  this.amount = 0;
}


}
